Makefile

```diff
@@ -78,14 +78,11 @@ get-kueue-image:
 	@rm -f .kueue_commit_id
 
 .PHONY: bundle-generate
-bundle-generate: operator-sdk regen-crd manifests get-kueue-image
-	@KUEUE_IMAGE=$$(cat .kueue_image); \
```

Comment:
Can you add some description to the GitHub PR? It looks like we are no longer using a kueue-0-11 image for bundle?

Reply:
I updated the PR description.  All the kueue images will get pushed to the same repo.  The digest will point to a build from kueue-0-11.